extension/src/popup/routes.tsx

```diff
@@ -21,4 +21,8 @@ export const routes: RouteObject[] = [
     path: 'open-wallet/connect-device',
     element: <ConnectDevicePage />,
   },
+  {
+    path: 'open-wallet/ledger',
```

Comment:
Do we have this path defined in two places now? Here and in common routes? If that is true we have to move other one away from commons. Other option is to remove this entry and use common routes only, but move disabled logic to one of validation function of FormLedger

Reply:
Did try to refactor this, but seems to brake a lot of things 😓 The way I expect this to work is that it overrides the path from the common route(last entry over existing one).

The way I see it, it is a clear separation, as web extension has different entry point. And it original idea, of adding routes, should stay the same?

@lukaw3d can you chip in here?

Reply:
How it can brake things? `'open-wallet/ledger'` defined in both routes.tsx files not in commons, or keeping route only in commons. Now with removed disabled"Somtehing" prop we go with commons approach. It means this entry here and FromLedgerWebExtension is not needed anymore, right?

Reply:
Ah I see now what you mean. The way I tried it was something like this:

```typescript
export function FromLedgerDynamic() {
  const href = useHref('/open-wallet/connect-device')
  const isExtension = runtimeIs === 'extension'

  const props = isExtension ? { webExtensionUSBLedgerAccess: () => openLedgerAccessPopup(href) } : {}

  return <FromLedger {...props} />
}
```

And import it in common only. And almost all unit tests started failing.